src/components/orders/AmountsDisplay/styled.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+
+import TokenImg from 'components/common/TokenImg'
+
+export const Wrapper = styled.div`
+  display: grid;
+  row-gap: 1rem;
+  justify-items: start;
+  align-items: center;
+  grid-template-columns: 10rem auto;
+  padding: 1.3rem 0;
+`
+
+export const RowTitle = styled.span`
+  display: flex;
+  align-items: center;
+  font-weight: ${({ theme }): string => theme.fontBold};
+
+  &::before {
+    content: '▶';
+    margin-right: 0.5rem;
+    color: ${({ theme }): string => theme.icon};
+    font-size: 0.75rem;
+  }
+`
+
+export const RowContents = styled.span`
+  display: flex;
+  align-items: center;
+
+  & > span:first-child {
+    margin: 0 0.5rem 0 0;
+  }
+`
+
+export const UsdAmount = styled.span`
+  color: ${({ theme }): string => theme.textPrimary1};
+  opacity: 0.5;
+`
+
+export const StyledImg = styled(TokenImg)`
+  width: 1.6rem;
+  height: 1.6rem;
```

Comment:
Also addressed on #209

Reply:
@alfetopito Ah those token names didn't render for me in the storybook and hadn't accounted for it. Will address this in this PR.

Reply:
Right, there was an issue with the storybook not rendering the links. Fixed on #210 
In the mean time you can try it out by starting the app (with `yarn start`) and using an existing order, like the ones I linked